### PR TITLE
Fix to handle timestamp fields with zeroed dates, quote table names w…

### PR DESCRIPTION
**Pull request by krakip**
…hen generating sequences

### CODE REVIEW

File: db_converter.py [hunk 1]
```diff
@@ -162,7 +162,7 @@ def parse(input_filename, output_filename):
                 if name == "id" and set_sequence is True:
                     sequence_lines.append("DROP SEQUENCE IF EXISTS %s_id_seq" % (current_table))
                     sequence_lines.append("CREATE SEQUENCE %s_id_seq" % (current_table))
-                    sequence_lines.append("SELECT setval('%s_id_seq', max(id)) FROM %s" % (current_table, current_table))
+                    sequence_lines.append("SELECT setval('%s_id_seq', max(id)) FROM \"%s\"" % (current_table, current_table))
```
Comment (michaelshobbs) [inline db_converter.py:165]: this seems unrelated to the timestamp fix

File: db_converter.py [hunk 2]
```diff
@@ -82,7 +82,7 @@ def parse(input_filename, output_filename):
                 creation_lines = []
             # Inserting data into a table?
             elif line.startswith("INSERT INTO"):
-                output.write(line.encode("utf8").replace("'0000-00-00 00:00:00'", "NULL") + "\n")
+                output.write(re.sub(r"'0000-00-00 00:00:00(\.[0]+)?'", "'1970-01-01 00:00:00'", line.encode("utf8")) + "\n")
```
Comment (michaelshobbs) [inline db_converter.py:85]: i'm worried this will break reports that expect a NULL in timestamp columns

**Comment by krakip** [comment]
I see you merged, but just to follow up on the comments:
- 0000-00-00 00:00:00 is treated differently from NULL in mysql, so the previous approach broke when the column was NOT NULL since the zero value is valid there but NULL is not.
- The other fix was indeed unrelated to timestamps. It has to do with sequences that are in tables with mixed case names.

Thanks for merging.

**Comment by michaelshobbs** [comment]
Makes sense. I wanted to make sure that anybody else creating reports wasn't filtering on these column types expecting `NULL`. We're all good though.
